test/cql-pytest/cassandra_tests/validation/operations/compact_storage_test.py

```diff
@@ -2117,19 +2117,6 @@ def testSelectDistinct(cql, test_keyspace):
                    row(2, 2),
                    row(1, 1))
 
-    # Test a 'wide row' thrift table.
```

Comment:
Please do not remove this test. It's a Cassandra unit test which is still valid. It is only a Thrift-like table but it does NOT use the thrift protocol, and should still pass. Unless we also want to deprecate WITH COMPACT STORAG, which is a different story.

Reply:
ack. will revert this change.